tests/python/gaia-ui-tests/gaiatest/apps/homescreen/app.py

```diff
@@ -14,18 +14,18 @@
 class Homescreen(Base):
 
     name = 'Default Home Screen'
-    manifest_url = '{}verticalhome{}/manifest.webapp'.format(Base.DEFAULT_PROTOCOL,Base.DEFAULT_APP_HOSTNAME)
```

Comment:
oh, probably, tag me ones that I should remove.  i rebased it this morning.  I'll rebase again but I wonder whether it'll put back old code back into gaia.